src/NMS.AMQP/Util/SymbolUtil.cs

```diff
@@ -39,7 +39,7 @@ public class SymbolUtil
         public static readonly Symbol OPEN_CAPABILITY_SOLE_CONNECTION_FOR_CONTAINER = new Symbol("sole-connection-for-container");
         public static readonly Symbol OPEN_CAPABILITY_ANONYMOUS_RELAY = new Symbol("ANONYMOUS-RELAY");
         public static readonly Symbol OPEN_CAPABILITY_DELAYED_DELIVERY = new Symbol("DELAYED_DELIVERY");
```

Comment:
this probably needs to be fixed to DELAYED-DELIVERY

Reply:
Need to cross check these with qpid again

Reply:
looks awkward, but seems no.

Reply:
https://www.oasis-open.org/committees/download.php/67638/amqp-bindmap-jms-v1.0-wd10.pdf in here its also _ instaead of -